ibis/expr/window.py

```diff
@@ -10,6 +12,29 @@ def _sequence_to_tuple(x):
     return tuple(x) if util.is_iterable(x) else x
 
 
+def _determine_how(preceding):
+    if isinstance(preceding, tuple):
+        start, end = preceding
+        if start is None:
+            offset_type = type(end)
+        else:
+            offset_type = type(start)
+    else:
+        offset_type = type(preceding)
+
+    if issubclass(offset_type, (int, np.integer)):
+        how = 'rows'
+    elif issubclass(offset_type, ir.IntervalScalar):
```

Comment:
This should actually be `ir.IntervalValue`, since a backend *could* support a column expression in the `preceding`/`following` argument.

Reply:
Actually, let's leave this as `ir.IntervalScalar` for now.